src-tauri/crates/eloelo_model/src/history.rs

```diff
@@ -16,6 +16,7 @@ pub struct HistoryEntry {
     pub timestamp: DateTime<Local>,
     pub winner: Vec<PlayerId>,
     pub loser: Vec<PlayerId>,
+    pub win_probability: f64,
```

Comment:
Maybe we shouldn't store that if this is to be derived from win scale and duration?